pr_testing/test-cmssw-provides.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash -xe
+if [ $# -ne 6 ]; then
+  echo Usage: test-cmssw-provides.sh scram_arch pkgtools_branch cmsdist_tag build_dir week_num cmssw_release
+  exit 1
+fi
+
+SCRAM_ARCH=$1
+PKG_TOOL_BRANCH=$2
+CMSDIST_TAG=$3
+BUILD_DIR=$4
+WEEK_NUM=$5
+CMSSW_RELEASE=$6
+
+SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"  # Absolute path to script
+CMS_BOT_DIR=$(dirname ${SCRIPTPATH})  # To get CMS_BOT dir path
+
+set +x
+eval $(scram unsetenv -sh)
+set -x
+
+rm -rf $WORKSPACE/test-provides
+mkdir -p $WORKSPACE/test-provides
+cd $WORKSPACE/test-provides
+if [ ! -d $WORKSPACE/pkgtools ]; then
+  git clone git@github.com:cms-sw/pkgtools --depth 1 -b $PKG_TOOL_BRANCH
+else
+  ln -s $WORKSPACE/pkgtools .
+fi
+
+if [ ! -d $WORKSPACE/cmsdist ]; then
+  git clone git@github.com:cms-sw/cmsdist --depth 1 -b $CMSDIST_TAG
+else
+  ln -s $WORKSPACE/cmsdist .
+fi
+
+sed -i -e "s!@release@!${WORKSPACE}/${CMSSW_RELEASE}!g" $CMS_BOT_DIR/pr_testing/cmssw-pr-package.spec
+
+# Check if we are testing with patch release
+pushd ${WORKSPACE}/${CMSSW_RELEASE}/src
+set +x
+eval $(scram runtime -sh)
+set -x
+FULL_RELEASE_BASE=$(echo ${CMSSW_FULL_RELEASE_BASE} | rev | cut -d '/' -f 1 | rev)
+if [ ${FULL_RELEASE_BASE} = ${CMSSW_RELEASE} ]; then
+  CMS_PKG_NAME='cmssw'
+else
+  CMS_PKG_NAME='cmssw-patch'
+fi
+set +x
+eval $(scram unsetenv -sh)
+set -x
+popd
+
+cp $CMS_BOT_DIR/pr_testing/cmssw-pr-package.spec cmsdist/
+
+sed -i -e "s!@provides@!$PROVIDELIST!" $CMS_BOT_DIR/pr_testing/cmssw-fake-package.spec
+cp $CMS_BOT_DIR/pr_testing/cmssw-fake-package.spec cmsdist/
+
+#mkdir -p test-provides/${SCRAM_ARCH}/var/lib
+# bootstrap cmsBuild
+pkgtools/cmsBuild --repo cms.week${WEEK_NUM} -a $SCRAM_ARCH -c cmsdist -i build --builders 1 -j 8 build cms-common
+
+RPM_CMD="/cvmfs/cms-ib.cern.ch/sw/$(uname -m)/week${WEEK_NUM}/common/cmspkg -a $SCRAM_ARCH env -- rpm"
+
+PROVIDELIST=$(${RPM_CMD} -q --provides cms+${CMS_PKG_NAME}+${CMSSW_RELEASE} --dbpath /cvmfs/cms-ib.cern.ch/sw/`uname -m`/week${WEEK_NUM}/${SCRAM_ARCH}/var/lib/rpm/)
```

Comment:
`--dbpath /cvmfs/cms-ib.cern.ch/sw/`uname -m`/week${WEEK_NUM}/${SCRAM_ARCH}/var/lib/rpm/` is not needed.

Reply:
Getting the provides is optional, this is only needed if we have local RPM DB. So this section should be in a conditional block

Reply:
Because we use `rpm` from cvmfs, right?